widgetastic_manageiq.py

```diff
@@ -1589,8 +1589,9 @@ def __repr__(self):
             params = ", ".join(["{}={}".format(attr, getattr(self, attr)) for attr in attrs])
             return "TimelinesEvent({})".format(params)
 
-    def __init__(self, parent, logger=None):
+    def __init__(self, parent, locator=None, logger=None):
         super(TimelinesChart, self).__init__(parent=parent, logger=logger)
+        self.locator = locator or '//div[contains(@class, "timeline-container")]'
```

Comment:
**Required**

It looks like the whole reason you've made this change is to support adding the chart with a `timeline-pf-chart` div.
(https://github.com/ManageIQ/integration_tests/pull/4627/files#diff-e25760b09cb1951870a11138d80a45f3R34)


It seems this chart element is patternfly, which means this change isn't needed for manageiq:
https://github.com/patternfly/patternfly-timeline/blob/27f0a72733238ebebf8ab69a5803be2ad7e6441c/src/timeline.js#L57

Unfortunately this is a major change if I'm right.

@mfalesni @izapolsk Thoughts? This should be added to patternfly and we need to bump that version before accepting this PR.